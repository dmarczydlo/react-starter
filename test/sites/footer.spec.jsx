import React from 'react';
import Footer from '../../src/layout/footer';

describe('Footer', () => {
    let wrapper;

    it('should be rendered', () => {
        wrapper = shallow(
            <Footer/>
        );
        expect(wrapper).toMatchSnapshot();
    });
});