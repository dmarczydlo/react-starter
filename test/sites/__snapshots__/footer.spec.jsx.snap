// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should be rendered 1`] = `
<div
  className="footer"
>
  (c) React starter footer
</div>
`;
